typings/blob-builder-defs.d.ts

```diff
@@ -0,0 +1,9 @@
+// Reference - https://developer.mozilla.org/en-US/docs/Web/API/BlobBuilder
+
+class BlobBuilder {
+  append: (data: ArrayBuffer) => void;
+  append: (data: Blob) => void;
+  append: (data: String, endings?: String) => void;
+  getBlob: (contentType?: String) => void;
+  getFile: (name: String, contentType?: String) => File;
+}
```

Comment:
Hmmm, I have just looked where we use this and I think we can remove `BlobBuilder` from our codebase entirely `new Blob` is supported among all the newer browsers (https://caniuse.com/#feat=blobbuilder), can you try to remove `BlobBuilder` from our codebase?

Reply:
Done!